Documentation/ApiOverview/Events/Events/FrontendLogin/_LogoutConfirmedEvent/_Services.yaml

```diff
@@ -0,0 +1,13 @@
+services:
+  _defaults:
+    autowire: true
+    autoconfigure: true
+    public: false
+
+  # [...]
+
+  # TODO: delete on dropping TYPO3 v12.4 support
```

Comment:
Remove it. It is still valid in v13. You don't have to remove this in v13.

Reply:
done